rt_diversity_filter.go

```diff
@@ -48,12 +51,15 @@ func (r *rtPeerIPGroupFilter) Allow(g peerdiversity.PeerGroupInfo) bool {
 	cpl := g.Cpl
 
 	if r.tableIpGroupCount[key] >= r.maxForTable {
-
+		dfLog.Debugw("rejecting (max for table) diversity of peer", g.Id, "cpl", g.Cpl, "ip group", g.IPGroupKey)
 		return false
 	}
 
 	c, ok := r.cplIpGroupCount[cpl]
 	allow := !ok || c[key] < r.maxPerCpl
+	if !allow {
+		dfLog.Debugw("rejecting (max for cpl) diversity of peer", g.Id, "cpl", g.Cpl, "ip group", g.IPGroupKey)
```

Comment:
I think you need a label for the ID:
```suggestion
		dfLog.Debugw("rejecting (max for cpl) diversity", "peer", g.Id, "cpl", g.Cpl, "ip group", g.IPGroupKey)
```